Resources/views/finish.html.twig

```diff
@@ -30,6 +30,12 @@
                 </div>
             {% endif %}
 
+            <p class="summary">
+                A summary of the repartition is available for download. Keep it secret, even for you, to not break Secret Santa spirit.<br />
+                Be aware that we do not persist anything forever so the summary will probably disappear in some time.<br />
```

Comment:
"so the summary is only available now, you will not be able to get it back later (as we do not keep anything persisted)."